README.md

```diff
@@ -233,71 +205,33 @@ See the [options](#options) section for information about flash message specific
 ## Displaying flash messages
 Then, to display somewhere in your app, add this to your template:
 
-```handlebars
-{{#each flashMessages.queue as |flash|}}
-  {{flash-message flash=flash}}
-{{/each}}
-```
-
-It also accepts your own template:
-
-```handlebars
-{{#each flashMessages.queue as |flash|}}
-  {{#flash-message flash=flash as |component flash|}}
-    <h6>{{component.flashType}}</h6>
-    <p>{{flash.message}}</p>
-    {{#if component.showProgressBar}}
-      <div class="alert-progress">
-        <div class="alert-progressBar" style="{{component.progressDuration}}"></div>
-      </div>
-    {{/if}}
-  {{/flash-message}}
-{{/each}}
-```
-
-### Styling with Foundation or Bootstrap
-By default, flash messages will have Bootstrap style class names. If you want to use Foundation, simply specify the `messageStyle` on the component:
-
-```handlebars
-{{#each flashMessages.queue as |flash|}}
-  {{flash-message flash=flash messageStyle='foundation'}}
-{{/each}}
+```hbs
+{{flash-messages flashes=flashMessages.queue}}
 ```
 
-### Sort messages by priority
-To display messages sorted by priority, add this to your template:
+To display messages sorted by priority, used `arrangedQueue`:
 
-```handlebars
-{{#each flashMessages.arrangedQueue as |flash|}}
-  {{flash-message flash=flash}}
-{{/each}}
+```hbs
+{{flash-messages flashes=flashMessages.arrangedQueue}}
 ```
 
-### Rounded corners (Foundation)
-To add `radius` or `round` type corners in Foundation:
-
-```handlebars
-{{#each flashMessages.arrangedQueue as |flash|}}
-  {{flash-message flash=flash messageStyle='foundation' class='radius'}}
-{{/each}}
-```
+It also accepts your own template. If the provided component isn't to your liking, you can easily create your own instead of using the contextual component:
 
-```handlebars
-{{#each flashMessages.arrangedQueue as |flash|}}
-  {{flash-message flash=flash messageStyle='foundation' class='round'}}
-{{/each}}
-```
-
-### Custom flash message component
-If the provided component isn't to your liking, you can easily create your own. All you need to do is pass in the `flash` object to that component:
-
-```handlebars
-{{#each flashMessages.queue as |flash|}}
-  {{custom-component flash=flash}}
-{{/each}}
+```hbs
+{{#flash-messages flashes=flashMessages.arrangedQueue as |flash|}}
+  {{#flash.component class=(concat "alert alert-" flash.type) click=(action flash.close) as |flashMessage|}}
```

Comment:
Foundation 6 merged alerts and panels into a 'callout' class, is 2.0 going to provide this support or should we handle it with our own component?

http://foundation.zurb.com/sites/docs/callout.html